pages.it/common/git-bisect.md

```diff
@@ -0,0 +1,20 @@
+# git bisect
+
+> Usa la ricerca binaria per trovare il commit che ha introdotto un bug.
+> Maggiori informazioni: <https://git-scm.com/docs/git-bisect>.
+
+- Avvia una sessione di bisect su un intervallo di commit limitato da un commit "cattivo", noto per contenere il bug, e un commit "buono", precedente al presentarsi del bug:
```

Comment:
like it! But I guess I'm going to replace `sessione di bisect` with `ricerca` in all the other descriptions, then

Reply:
However, I'd keep the association "cattivo" in it, because it's going to be used with no real explanation in the next description - when it'll come to label commits. Something like:
`Avvia una ricerca su un intervallo di commit definito da un commit "cattivo" contenente il bug ed un altro commit "buono" privo del bug (solitamente precedente al cattivo):`

Reply:
Sure, something like this?

> - Avvia una ricerca su un intervallo di commit definito dal commit "cattivo" contenente il bug ed un altro commit "buono" privo del bug (solitamente più vecchio):